package excecao;

public class ChecadaVsNChegada {

    public static void main(String[] args) {

	try {
	    geraErro1();
	} catch (RuntimeException e) {
	    System.out.println(e.getMessage());
	}

	geraErro2();

	System.out.println("Fim ?");
    }

    // NÃ£o checada
    static void geraErro1() {
	throw new RuntimeException("Ocorreu um erro #01");
    }

    //Chegada
    static void geraErro2() throws Exception{
	throw new Exception("Ocorreu um erro #02");
    }
}
